Simplified.xcodeproj/project.pbxproj

```diff
@@ -1536,6 +1547,28 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXCopyFilesBuildPhase section */
+		177C25FD28B704A600A786F1 /* Embed Frameworks */ = {
```

Comment:
hmm is this for the noDRM and R2 targets? Or maybe the unit tests... I thought I already added NYPLUtilities to the main SimplyE and OE targets.

Reply:
This is for the unit tests targets